test-network-nano-bash/configtx.yaml

```diff
@@ -343,45 +268,96 @@ Channel: &ChannelDefaults
 #
 ################################################################################
 Profiles:
-
-    TwoOrgsOrdererGenesis:
-        <<: *ChannelDefaults
-        Orderer:
-            <<: *OrdererDefaults
-            Organizations:
-                - *OrdererOrg
-            Capabilities:
-                <<: *OrdererCapabilities
-        Consortiums:
-            SampleConsortium:
-                Organizations:
-                    - *Org1
-                    - *Org2
-    TwoOrgsChannel:
-        Consortium: SampleConsortium
-        <<: *ChannelDefaults
-        Application:
-            <<: *ApplicationDefaults
-            Organizations:
-                - *Org1
-                - *Org2
-            Capabilities:
-                <<: *ApplicationCapabilities
-    Org1Channel:
-        Consortium: SampleConsortium
-        <<: *ChannelDefaults
-        Application:
-            <<: *ApplicationDefaults
-            Organizations:
-                - *Org1
-            Capabilities:
-                <<: *ApplicationCapabilities
-    Org2Channel:
-        Consortium: SampleConsortium
-        <<: *ChannelDefaults
-        Application:
-            <<: *ApplicationDefaults
-            Organizations:
-                - *Org2
-            Capabilities:
-                <<: *ApplicationCapabilities
+  TwoOrgsChannel:
```

Comment:
Both profiles have two application orgs so it doesn't really help distinguish by calling it "TwoOrgsChannel". How about something like "ChannelUsingRaft" and "ChannelUsingBFT".